### ISSUE TITLE
Wi-Fi Client Mode support for on phone Hotspot

### ISSUE THREAD

**Issue by Swap-File** [cpebit/aawireless-issues#170]
I would like to suggest the feature of Wi-Fi Client Mode support for a hotspot running on the phone where AAWireless is running.

It appears that AAWireless (the device) can connect to my phone's hotspot when in client mode, but AAWireless (the app) can't find the device.

When I asked in the XDA support forum, it was mentioned that this is currently expected behavior since client mode only supports external hot spots (currently). 

Thanks!

**Comment by Snirpo** [comment]
Yes, this is possible, but a bit low prio for now. It's on our list though.

**Comment by DonvdH** [comment]
I just switched from a Samsung S20 to the Google Pixel 7 Pro and it appears that the Android Auto connection doesn't work when the phone hotspot is enabled.

I hoped to be able to use the client mode as a workaround, but apparently no android auto connection gets established after it connects to the phone hotspot.. It would be great if this feature gets implemented, it seems I need to revert to using a cable in the meantime untill either Google fixes the Pixel 7 Pro firmware or this feature gets implemented.
